sycl/test/group-algorithm/broadcast.cpp

```diff
@@ -0,0 +1,70 @@
+// RUN: %clangxx -fsycl -D __SYCL_INTEL_GROUP_ALGORITHMS__ %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+//==------------- broadcast.cpp - SYCL group broadcast test --*- C++
+//-*-----------==//
```

Comment:
Formatting seems to be off.
NOTE: according to my understanding LLVM project does not require adding copyright header to test sources, so you can fix it by removing lines 6-13.